### PR TITLE
fixing bugs within the SuperCollider Quartz composer binding documentation

### PR DESCRIPTION
**Pull request by tmm88**
some of the patches names have been changed. one patch has been removed. i commented the examples of the missing patch, and corrected the files that were wrongly spelled

### CODE REVIEW

**Comment by tiagmoraismorgado** [comment]
just added one further example

File: HelpSource/Classes/QuartzComposerView.schelp [hunk 1]
```diff
@@ -193,7 +212,7 @@ b = Button(w, Rect(0, 0, 150, 20))
     .states_([["pick another QC file"]])
     .action_({ File.openDialog("", { |path| m.path_(path) }) });
 m = QuartzComposerView(w, Rect(0,20,400, 260));
-m.path = Platform.helpDir +/+ "QC/ImageFilter.qtz";
+m.path = Platform.userAppSupportDir +/+ "Help/QC/ImageFilter.qtz"; // file missing
```
Comment (brianlheim) [inline HelpSource/Classes/QuartzComposerView.schelp]: if the file is gone, just delete the example, don't leave a non-working example in the documentation

File: HelpSource/Classes/QuartzComposerView.schelp [hunk 2]
```diff
@@ -317,26 +336,4 @@ Event.addEventType(\quartz, { |server|
...
-Pbind(
-	\type, \quartz,
-	\view, m,
-	\key, \level,
-	\value, Pwhite(0, 1.0, inf).explin(0.01,1.01, 0, 1),
-	\dur, 0.1
-).play;
-)
-
-m.setInputValue(\level, 0.5);
```
Comment (brianlheim) [inline HelpSource/Classes/QuartzComposerView.schelp]: why did you delete this section?

**Comment by brianlheim** [comment]
@tiagmoraismorgado - please add useful commit messages. Our [contributing.md](https://github.com/supercollider/supercollider/blob/master/CONTRIBUTING.md#making-changes) has a section about this.

**Comment by muellmusik** [comment]
@tiagmoraismorgado first of all thanks very much for taking this on!

I've attached the missing qtz files, so I'd suggest restoring the examples.


[missingQCstuff.zip](https://github.com/supercollider/supercollider/files/1220610/missingQCstuff.zip)

**Comment by tiagmoraismorgado** [comment]
you're welcome scott. as a supercollider user and a quartz composer user, i find this feature a huge advantage. and many people were pointing me out that this had been deprecated. fortunately with your help i was able to figure out that the problem was solely on the paths of the files, so i decided to fix this (although i would find it more efficient to fix solely the helpDir method of plattform, because that would fix the problem with a single line of code, but if @brianlheim  suggested to do it like fixing the other way around, i think it's preferrable to do it as he says. tell us what you think)

**Comment by tiagmoraismorgado** [comment]
@brianlheim can you help within uploading the missing quartz composer files scott just posted here? thanks

**Comment by brianlheim** [comment]
@tiagmoraismorgado take care of your own PRs please.

**Comment by telephon** [comment]
(in other words, @tiag, you can do that yourself without problem)

File: HelpSource/Classes/QuartzComposerView.schelp [hunk 3]
```diff
@@ -66,14 +66,33 @@ Open the currently loaded composition in Quartz Composer. You will need to reloa
 EXAMPLES::
 
 code::
-////////// Simple example
+
+////////// Simple example A (fullscreen)
+(
+var width= 1500, height= 1250; // use a resolution higher then the one of your screen
```
Comment (snappizz) [inline HelpSource/Classes/QuartzComposerView.schelp]: whitespace issues here and on subsequent lines

i don't know anything about QuartzComposerView, but this seems like a pretty crude approach to fullscreen doesn't it? is there a better way?

Comment (joshpar) [inline HelpSource/Classes/QuartzComposerView.schelp]: This gives us our screen bounds. Please use something like that.

(
var screenBounds, width, height;
screenBounds = Window.screenBounds;
width = screenBounds.width;
height = screenBounds.height;
[screenBounds, width, height].postln;
)

Comment (snappizz) [reply HelpSource/Classes/QuartzComposerView.schelp]: still seems hacky, is there a way to make a maximized window?

**Comment by brianlheim** [comment]
@tmm88 do you plan to continue work on this?

**Comment by brianlheim** [comment]
Closing this stale PR
